src/test/ui/rfc-2565-param-attrs/param-attrs-proc-macro.rs

```diff
@@ -0,0 +1,10 @@
+// aux-build:param-attrs-proc-macro-aux.rs
+// run-pass
+
+#![feature(param_attrs)]
+
+extern crate param_attrs_proc_macro_aux;
+
+fn foo(#[param_attrs_proc_macro_aux::bar] baz: i32) {}
```

Comment:
This is an error, no?
The error should say something like "attribute `param_attrs_proc_macro_aux::bar` is currently unknown" right now.